components/QuickStartMenu.tsx

```diff
@@ -24,6 +67,62 @@ const QuickStartMenu = ({ className }: QuickStartMenuProps): JSX.Element => {
           </g>
         </svg>
       </button>
+      {/* NOTE:: Dropdown currently absolutely positioned. Revist if used in place other than the navigation */}
+      <div
+        ref={quickStartMenuRef}
+        className={classnames(
+          'shadow-sm',
+          'bg-white',
+          'border',
+          'border-gray-light',
+          'absolute',
+          'top-20',
+          'w-56',
+          'right-1',
+          'lg:transform-none',
+          'lg:transition-none',
+          'transform-gpu',
```

Comment:
Are these `transform` and `transition` utilities being actually used for anything here? If the element is going from `display: none` to `display:whatever` - what's the purpose?